src/content/guides/using-persistent-volumes-on-aws-with-efs-csi-driver/index.md

```diff
@@ -0,0 +1,140 @@
+
+1. Open the [AWS management](https://aws.amazon.com/console/) console in the account your cluster is located.
+2. Select EFS from the services list.
+3. Create a new file system and select the VPC where your cluster is located.
+4. Select the Availability Zone with the subnets your instances are located on and the security-groups of your node pools.
+5. Choose the throughput and performance mode, no file system policy or access points are needed.
+6. Create the instance and note the EFS instance id.
+
+## Installing the EFS CSI driver
+
+To install the EFS CSI driver you will need to follow these steps:
+
+1. Access the [web interface](/reference/web-interface/) and select the cluster on which you want to install the EFS CSI driver.
+2. Open the Apps tab.
+3. Click the Install App button
+4. Select the Giant Swarm Playground catalog.
+5. Select the App named aws-efs-csi-driver.
+6. Click the Configure & Install button. Make sure that the correct cluster is selected.
+7. Click the Install App button.
+
+## Storage classes
+
+Once you installed the `aws-efs-csi-driver` from _Giant Swarm Playground_ catalog, Kubernetes cluster will have a storage class `efs-csi` deployed.
+
+## Deploy a sample application
+
+In order to see if all works out we need to create some manifests:
+
+First we apply a PersistentVolume with a capacity of 5 GB storage, here you need to provide EFS instance id:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: efs-pv
+spec:
+  capacity:
+    storage: 5Gi
+  volumeMode: Filesystem
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: efs-csi
+  csi:
+    driver: efs.csi.aws.com
+    volumeHandle: fs-xxxxxxxx
+```
+
+**Note:** Storage capacity is a required field, you need to specify a valid value but is not used when creating the file system. It's important to note EFS is an elastic file system, it does not enforce any file system capacity limits.
+
+Additionally we need a PerstentVolumeClaim with a request of 5 GB storage:
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: efs-claim
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: efs-csi
+  resources:
+    requests:
+      storage: 5Gi
+```
+
+And finally, we apply a Pod which writes every 5 seconds the current date in UTC time in a file located in our EFS storage mount:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: efs-app
+spec:
+  containers:
+  - name: linux
+    image: amazonlinux:2
+    command: ["/bin/sh"]
+    args: ["-c", "while true; do echo $(date -u) >> /efs-data/out.txt; sleep 5; done"]
+    volumeMounts:
+    - name: efs-storage
+      mountPath: /efs-data
+  volumes:
+  - name: efs-storage
+    persistentVolumeClaim:
+      claimName: efs-claim
+```
+
+**Warning:**
+By default, new EFS file systems are owned by root:root. You might need to change file system permissions if your container is not running as root.
```

Comment:
```suggestion
**Warning:** By default, new EFS file systems are owned by root:root. You might need to change file system permissions if your container is not running as root.
```

Will readers know how to do that, or can we make it easier for them?

Reply:
I added more context how you can change permissions by using EFS access points.